apps/web/playwright/filters.e2e.ts

```diff
@@ -0,0 +1,696 @@
+      data: {
+        formFillerId: "test-filler-1",
+        formId: form.id,
+        response: {
+          name: {
+            label: "Name",
+            value: "John Doe",
+          },
+        },
+        routedToBookingUid: booking1.uid,
+      },
+    });
+
+    await prisma.app_RoutingForms_FormResponse.create({
+      data: {
+        formFillerId: "test-filler-2",
+        formId: form.id,
+        response: {
+          name: {
+            label: "Name",
+            value: "Jane Smith",
+          },
+        },
+        routedToBookingUid: booking2.uid,
+      },
+    });
+
+    await page.goto(`/insights/routing`);
+
+    await applyFilter(page, "bookingUserId", user1.name || "User One");
+
+    await expect(page.getByText("John Doe")).toBeVisible();
+    await expect(page.getByText("Jane Smith")).toBeHidden();
+
+    await prisma.booking.delete({ where: { id: booking1.id } });
+    await prisma.booking.delete({ where: { id: booking2.id } });
+  });
+
+  test("bookingAttendees filter: should filter by attendee name or email", async ({
+    page,
+    users,
+    routingForms,
+    prisma,
+  }) => {
+    const owner = await users.create(undefined, {
+      hasTeam: true,
+      isUnpublished: true,
+      isOrg: true,
+      hasSubteam: true,
+    });
+    await owner.apiLogin();
+    const membership = await owner.getOrgMembership();
+
+    const formName = "Attendee Filter Test Form";
+    const form = await routingForms.create({
+      name: formName,
+      userId: owner.id,
+      teamId: membership.teamId,
+      fields: [
+        {
+          type: "text",
+          label: "Name",
+          identifier: "name",
+          required: true,
+        },
+      ],
+    });
+
+    const booking1 = await prisma.booking.create({
+      data: {
+        uid: `booking-${uuidv4()}`,
+        title: "Test Booking 1",
+        startTime: new Date(),
+        endTime: new Date(Date.now() + 60 * 60 * 1000),
+        userId: owner.id,
+        status: BookingStatus.ACCEPTED,
+        attendees: {
+          create: [
+            {
+              email: "john.doe@example.com",
+              name: "John Doe",
+              timeZone: "UTC",
+            },
+          ],
+        },
+      },
+    });
+
+    const booking2 = await prisma.booking.create({
+      data: {
+        uid: `booking-${uuidv4()}`,
+        title: "Test Booking 2",
+        startTime: new Date(),
+        endTime: new Date(Date.now() + 60 * 60 * 1000),
+        userId: owner.id,
+        status: BookingStatus.ACCEPTED,
+        attendees: {
+          create: [
+            {
+              email: "jane.smith@example.com",
+              name: "Jane Smith",
+              timeZone: "UTC",
+            },
+          ],
+        },
+      },
+    });
+
+    await prisma.app_RoutingForms_FormResponse.create({
+      data: {
+        formFillerId: "test-filler-1",
+        formId: form.id,
+        response: { name: { label: "Name", value: "Response 1" } },
```

Comment:
IIRC, key of response object should be fieldId. Check if in correct. If it is, update these accordingly.